runtime/metricsview/query.go

```diff
@@ -203,6 +203,10 @@ func (t TimeGrain) Valid() bool {
 	return false
 }
 
+func (t TimeGrain) isLess(tg TimeGrain) bool {
```

Comment:
Not sure if the verb "less" makes sense in the context of grains. Maybe is `IsSmaller` instead?